Dockerfile

```diff
@@ -31,16 +31,27 @@ RUN mkdir -p /app/entities
 
 ################# DEVELOPMENT ####################################
 FROM base as development
-COPY . .
+COPY wsgi.py requirements.txt requirements_dev.txt plugins.yml ./
+COPY app app
+COPY entities entities
+COPY python_mapping_plugins python_mapping_plugins
 
 # Run static security check and linters
 RUN pip install -q --trusted-host pypi.org --trusted-host files.pythonhosted.org -r requirements_dev.txt
-RUN pre-commit run --all-files  \
-  && safety check -r requirements.txt -r requirements_dev.txt
+#RUN pre-commit run --all-files  \
+#  && safety check -r requirements.txt -r requirements_dev.txt
 
 # Run pytest with code coverage
 # RUN pytest --cov app
 
+# For debugging: install oteapi locally
+#COPY oteapi-core oteapi-core
```

Comment:
The local oteapi-core should not be inside oetapi-services. Copy from path to is my suggstion.
```suggestion
# copy oteapi-core from local path
#COPY ../oteapi-core oteapi-core
```
Furthermore, this does not allow for runing services locally while developing oteapi-core. That would require  mounting the oteapi-core as a volume and pip installing from there. 

I have tried to do that but right now the services fail as soon as there is an error, making everything stop.

Reply:
Agree, placing oteapi-core inside oteapi-services is not the correct way to do things. It is just a hack that makes it much easier to debug issues within oteapi-core that makes the services crash.